lib/renderer/webview/webview.h

```diff
@@ -2298,8 +2308,22 @@ struct webview_priv
                     }];
       if ([NSApp runModalForWindow:panel] == NSModalResponseOK)
       {
-        const char *filename = [[[panel URL] path] UTF8String];
-        strlcpy(result, filename, resultsz);
+        /* https://stackoverflow.com/a/15318065/6748719 */
+        int filesCount = [[panel URLs] count];
+        char **filePaths = (char **)malloc(sizeof(char *) * (filesCount + 1));
+        int i;
+        for (id url in [panel URLs]) {
+          const char *filePath = [[url path] UTF8String];
+          int len = strlen(filePath);
+          char *filePathCopy = (char *)malloc(sizeof(char) * (len + 1));
+          strcpy(filePathCopy, filePath);
+          filePaths[i] = filePathCopy;
+          i++;
+        }
+        filePaths[i] = NULL;
+
+        memcpy(result, filePaths, resultsz);
+        memcpy(resultlen, &filesCount, sizeof(filesCount));
```

Comment:
An alternative approach to this is just have a string that uses a `\n` delimiter to split the filenames up. You just keep appending to it.

Reply:
I agree `\n` separated strings is simple solution. But AFAIK, `\n` valid character inside filename. Anyway I will go with your suggestion cuz i don't think people use `\n` inside filenames